#167719918 fix user interface design ui

WinnersProx opened the pull request:
### #167719918 remove useless link and fix some pages ui

#### What does this PR do?
improve user interface design for some pages
#### Description of Task to be completed?
improve these pages: fix landing page, fix login page, improve signup page, improve trips page
#### How should this be manually tested?
Test one of these features [here](https://winnersprox.github.io/way-farer-app/).
#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/167719918
#### Screenshots
![capture](https://user-images.githubusercontent.com/34940230/62505032-d7d91a00-b7af-11e9-95e4-d35398c69920.PNG)
![login](https://user-images.githubusercontent.com/34940230/62533137-d4b74b80-b7fa-11e9-8797-9899c9bbb2d5.PNG)
![landing](https://user-images.githubusercontent.com/34940230/62505204-749bb780-b7b0-11e9-9edc-fe1cfd840120.PNG)

WinnersProx committed:
bug(UI): improve UI
fix landing page
fix login page
improve signup page
improve trips page

[finishes #167719918]

WinnersProx committed:
bug(UI): improve UI
fix landing page
fix login page
improve signup page
improve trips page

[finishes #167719918]

Oluwasegun-AA commented:
@WinnersProx, the Navbar on the login page should not have `Bookings` because as a user who has not logged in, you are not expected to have any booking history.
please fix this and merge

Oluwasegun-AA commented:
@WinnersProx, also correct your PR title. let it be in line with the guide paste in the slack channel

WinnersProx committed:
bug(Login): remove useless link
remove bookings from login page
remove bookings from login navbar
improve signing in as admin
[finishes #167719918]

WinnersProx commented:
already fixed and merged

On Tue, Aug 6, 2019 at 1:30 AM Adepoju Oluwasegun <notifications@github.com>
wrote:

> @WinnersProx <https://github.com/WinnersProx>, also correct your PR
> title. let it be in line with the guide paste in the slack channel
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/WinnersProx/way-farer-app/pull/16?email_source=notifications&email_token=AIKSKRROE5PBRQBV2AAALFDQDEZARA5CNFSM4IJQEFJ2YY3PNVWWK3TUL52HS4DFVREXG43VMVBW63LNMVXHJKTDN5WW2ZLOORPWSZGOD3ULNSY#issuecomment-518567627>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AIKSKRRAVRKLLOAZU5SGWA3QDEZARANCNFSM4IJQEFJQ>
> .
>
